api/src/main/java/org/openmrs/api/db/hibernate/HibernateUserDAO.java

```diff
@@ -43,6 +43,8 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+import static org.openmrs.util.OpenmrsConstants.ZERO_LOGIN_ATTEMPTS_VALUE;
```

Comment:
Our convention has not been to import static. Can we do it like we have done else where for OpenmrsConstants?